### ISSUE TITLE
Make user field multiple in AccountEligibility

### ISSUE THREAD

**Issue by lilients** [Pathoschild/Wikimedia-contrib#152]
We would like to check a list of users for a specific event in AccountEligibility. The easiest way to do this would be, if we could make the input field for the user name a textfield where we can enter a list of usernames. Would that be something useful for the tool in general?

**Comment by Pathoschild** [comment]
Yep, that'd be useful for a lot of events. To avoid confusion, I think the default UI should still be one name like before; we could maybe have a one-line textarea that expands if you hit enter or paste multiple lines, or add a link to enable multi-user input?

**Comment by lilients** [comment]
Yes, maybe a separate view for multiple user check would be useful. I guess the result presentation would also look a little bit different. I will play around with this a bit and see what works. :)

**Comment by lilients** [comment]
@Pathoschild can you add me to the project at toolforge, please? :sun_with_face:

**Comment by Pathoschild** [comment]
@lilients Are you interested in becoming a permanent maintainer for Accounteligibility? If you just want to set up a test environment, there are [instructions in the readme](https://github.com/Pathoschild/Wikimedia-contrib#for-maintainers) to set it up with any Toolforge account.

**Comment by lilients** [comment]
@Pathoschild Ah, thanks for the hint. I did indead misread the readme. :blush:  In that case I do not need a maintainer access. :sweat_smile:
